saw-remote-api/python/saw/__init__.py

```diff
@@ -91,6 +91,22 @@ def is_success(self) -> bool:
         return False
 
 
+@dataclass
+class AssumptionSucceeded(VerificationResult):
```

Comment:
Currently, `AssumptionFailed` is a subclass of `VerificationFailed`. By analogy, should `AssumptionSucceeded` be a subclass of `VerificationSucceeded`? If so, that would permit simplifying the implementation of `__init__` by simply invoking `super().__init__(...)`.

Reply:
Oh, good point! I'll do that.